code/modules/events/infestation.dm

```diff
@@ -131,6 +135,14 @@
 	for(var/spawned_mob in chosen_mob_types)
 		new spawned_mob(chosen_area.random_space())
 
+/// Sends out a message to all IPCs on the ship warning them that hivebots are present.
+/datum/event/infestation/proc/hivebot_message()
+	for(var/mob/living/carbon/human/H in GLOB.living_mob_list) {
```

Comment:
Use `player_list` for a list of all mobs with a client (excluding new players), it's shorter to traverse compared to a list of all the mobs that are alive, and saves you from the logical error you did below of not checking if the mob has a client before sending it a message (thus generating proc calls for no value)

Also, no C-style syntax, remove all the `{` and `}` and use the indentation as Python (and the entire rest of this codebase, aside from macros)